src/Tribe/Main.php

```diff
@@ -683,7 +683,7 @@ public function bind_implementations(  ) {
 			tribe_register_provider( TEC\Events\Blocks\Controller::class );
 
 			// Site Editor
-			tribe_register_provider( TEC\Events\Editor\Full_Site\Controller::class );
```

Comment:
Do we need to deprecate any of these old clases?

Reply:
I don't think it's necessary, they are very new. @Camwyn or @bordoni ?

Reply:
I think we should to be honest. Anything public might be in use _somewhere_, and it covers us if we missed something.

Reply:
I don't want to register both - does the deprecated need to be registered as well?

Reply:
no